scripts/plot_frame_times.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# This script can be used to plot frame time layer output with gnuplot. A PNG is
+# generated containing a CDF of frame times in milliseconds.
+#
+# Usage:
+#   plot_fram_times.sh frame_time_a.txt frame_time_b.txt
```

Comment:
Can you mention that what the default output filename is?

Reply:
Done